security-force-monitor/sfm-cms
# Change incident:perpetrator_classification to a list type

## Overview

Change `incident:perpetrator_classification` to allow it to store a list of values.

Connects #692.

## Testing Instructions

* Run a quick data import (this will take ~5 minutes):

```
docker-compose run --rm app ./manage.py import_google_doc --source_doc_id 1e_N1oWUYFasm7FL1N6w_gKZ4G3NwJDc95xJtz0NL3DE --doc_id 1DOKkGmcUX5o0DJXlP1LzT54PNvspDdpd6fOK5igx7Dg
```

* Remake the search index with `docker-compose run --rm app ./manage.py make_search_index --recreate` (this will take ~10 minutes)
* Start the development server with `docker-compose up`
* Visit http://localhost:8000/en/search/?entity_type=Violation, try the perpetrator classification facet, and confirm that the filtering and display of perpetrator classifications still works as you would expect
* Visit http://localhost:8000/en/violation/view/103df674-22df-4d34-8166-57abdb25bbfe/ and confirm you see multiple perpetrator classifications in the jumbotron
* Visit http://localhost:8000/en/violation/edit/103df674-22df-4d34-8166-57abdb25bbfe/, add a perpetrator classification to the list, and confirm it gets saved properly
